src/extension/publish/Library.js

```diff
@@ -114,6 +114,29 @@ const Library = function(data)
     });
 
     let self = this;
+    let timelineNames = data.Timelines.map(timelineData => timelineData.name);
+    let graphics = 0;
+    for(let timelineData of data.Timelines){
```

Comment:
There’s probably a cleaner way to write this using filter, map, and forEach. But it works